documentation/index.md

```diff
@@ -0,0 +1,25 @@
+Charmed Superset is an open-source, production-ready data exploration and visualization platform based on [Apache Superset<sup>TM</sup>](https://superset.apache.org/).
+
+Superset can replace or augment proprietary business intelligence tools. It integrates with a variety of data sources and provides a powerful SQL editor for advanced querying as well as a low-code interface for building charts and dashboards.
+
+Charmed Superset meets the needs of providing data insights in a repeatable, governable and flexible manner. Contributing to higher data literacy and data driven business decisions.
+
+It is intended for data practitioners who require advanced visualization without the complexity of deployment and on going management of the service.
+
+## In This Documentation
+
+|                                                                                                                                                                                                     |                                                                                                                                                                     |
+| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Tutorial </br> **Get started** - a hands-on [introduction to Charmed Superset](tutorial/01-introduction.md) for new users </br> | How-to guides </br> **Step-by-step guides** covering key operations and common tasks, such as [performance](how-to/optimize-deployment-performance.md) and [observability](how-to/observe-superset-metrics.md) |
+| Reference </br> **Technical information**, including [supported feature flags](reference/feature-flags.md) and [version compatibility](reference/revision-resource-compatibility.md)                                                                              | Explanation </br> **Concepts** - discussion and clarification of key topics     |
+
+
+## Project and Community
+
+Charmed Superset is a member of the Ubuntu family. It’s an open source
+project that warmly welcomes community projects, contributions, suggestions,
+fixes and constructive feedback.
+
+- [Read our Code of Conduct](https://ubuntu.com/community/code-of-conduct)
+- [Join the Discourse forum](https://discourse.charmhub.io/tag/temporal)
+- [Contribute and report bugs](https://github.com/canonical/temporal-k8s-operator)
```

Comment:
```suggestion
- [Contribute and report bugs](https://github.com/canonical/temporal-k8s-operator).
```

Reply:
You can also add links to Matrix channels, project examples or any other relevant community link (if applicable).